packages/cli/src/databases/entities/insights-by-period.ts

```diff
@@ -0,0 +1,62 @@
+import { BaseEntity, Column, Entity, PrimaryGeneratedColumn } from '@n8n/typeorm';
+import { UnexpectedError } from 'n8n-workflow';
+
+import { datetimeColumnType } from './abstract-entity';
+import type { PeriodUnits } from './insights-shared';
+import {
+	isValidPeriodNumber,
+	isValidTypeNumber,
+	NumberToPeriodUnit,
+	NumberToType,
+	PeriodUnitToNumber,
+	TypeToNumber,
+} from './insights-shared';
+
+@Entity()
+export class InsightsByPeriod extends BaseEntity {
```

Comment:
as pointed above, we should be able to move this into the module folder as well.